frontend/test/lit/test_decomposition.py

```diff
@@ -27,7 +27,7 @@
 def get_custom_device_without(num_wires, discards):
     """Generate a custom device without gates in discards."""
 
-    lightning = qml.device("lightning.qubit", wires=3)
+    lightning = qml.device("lightning.kokkos", wires=3)
```

Comment:
It just uses kokkos list of operations because the one from lightning is a frozen set.